### DISCUSSION TITLE
Ability to create a folder-like structure

### DISCUSSION THREAD

**Discussion by lukaszzyla** [jonaswinkler/paperless-ng#1232] [Feature Requests]
Sometimes it is easier to do general grouping of files in a directory tree.
It would be nice to have the ability to create some folders within Documents tab,

**Comment by wave864** [comment]
This would be a great feature. So far I have been using a desktop literature db to archive my correspondence and one thing I do miss in Paperless NG is the ability to create folder structures like

```
Rental Appartment
|- Electricity
|- Heat
|- Insurance
   |- Fire
   |- Tornados

```
In combination with the tab system this would allow for a lot of flexibility (and take some weigt of the "correspondent" field at the same time).

**Reply by ma651851384** [reply]
Hi @lukaszzyla  and @wave864 . Is the tagging of the document not the solution for your use case? Because you can filter for any tag you would like to, also more than one tag. In case of the example of @wave864 you would use "fire" and "insurance" as tags for a document. If you choose "Insurance" as tag, you would see all documents with "Fire", "Tornados". But if you choose additionally the tag  "Fire", you would only see the "Fire" docs.

The usage of tags makes the organisation easier because it does not have the limitaion like "folder already exists" in the same folder level or other disadvantages. For instance: Should I organise a bill from a shop in folder structure "bills/shopname" or "shopname/bills"? That doen't matter with tags.

I personally also have to learn to not think in "folders" but instead, to use the power of tags :-) We are using it daily in normal file systems but with that limitations...

Regards,
